skimage/morphology/watershed.py

```diff
@@ -120,19 +120,36 @@ def _validate_connectivity(image_dim, connectivity, offset):
     return c_connectivity, offset
 
 
-def _compute_neighbors(image, structure, offset):
-    """Compute neighborhood as an array of linear offsets into the image.
+def _offsets_to_raveled_neighbors(image_shape, selem, center):
```

Comment:
@jni I took the liberty to merge the old and new function. I think that the new names are a little clearer and more documentation shouldn't hurt either.